modules/nw-configuring-route-timeouts.adoc

```diff
@@ -7,26 +7,29 @@
 [id="nw-configuring-route-timeouts_{context}"]
 = Configuring route timeouts
 
-You can configure the default timeouts for an existing route when you
-have services in need of a low timeout, which is required for Service Level
-Availability (SLA) purposes, or a high timeout, for cases with a slow
-back end.
+You can configure the default timeouts for an existing route when you have services in need of a low timeout, which is required for Service Level Availability (SLA) purposes, or a high timeout, for cases with a slow back end.
+
+[IMPORTANT]
+====
+If you configured a user-managed load balancer for your network, ensure that the timeout value for the user-managed load balancer is higher than the timeout value for the route service. This configuration prevents network congestion issues over the network.
```

Comment:
I'm not sure if `user-managed` load balancer is clear enough, from the description of the bug maybe `user-managed external load-balancer in front of OpenShift` is better ?